Components/modal.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PlayX-modal</title>
+    <link rel="stylesheet" href="../Styles/modal.css" />
+    <link rel="stylesheet" href="../Styles/main.css" />
+    <script src="https://kit.fontawesome.com/fa50d5d333.js" crossorigin="anonymous"></script>
```

Comment:
why script is used for font awesome ?. Font awesome can be imported using link.

Reply:
don't know how to do using only links. but sure  read about it

Reply:
Please go through this link and get the correct version of whatever you've used in this project.
https://cdnjs.com/libraries/font-awesome

Reply:
thanks will do that